pkg/scheduler/api/resource_info.go

```diff
@@ -418,10 +418,10 @@ func (r *Resource) LessEqualWithReason(rr *Resource, defaultValue DimensionDefau
 	}
 
 	if !lessEqualFunc(r.MilliCPU, rr.MilliCPU, minResource) {
-		return false, "Insufficient cpu"
```

Comment:
Suggest not add any prefix for the resource for this is only a common used tool function. The prefix may narrow the usage.

Reply:
The origin prefix is reason, which is also not common, `LessEqual` func has many usage, it will cause a big change if wechange this function directly.